hunting/windows/docs/suspicious_dns_txt_record_lookups_by_process.md

```diff
@@ -16,1577 +16,14 @@ Adversaries may leverage DNS TXT queries to stage malicious content or exfiltrat
 ## Query
 
 ```sql
-f```
```

Comment:
This diff was caused by not having the query as an array. This was fixed and the docs regenerated.